src/api/spec/controllers/webui/staging_workflows/request_ignorations_controller_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe Webui::StagingWorkflows::RequestIgnorationsController, type: :controller do
+  let(:user) { create(:confirmed_user, login: 'tom') }
+  let(:project) { user.home_project }
+  let(:factory) { create(:project, name: 'openSUSE:Factory') }
+  let(:staging_workflow) { project.create_staging }
+
+  let(:source_package) { create(:package) }
+  let(:target_package) { create(:package, name: 'target_package', project: factory) }
+  let(:bs_request) do
+    create(:bs_request_with_submit_action,
+           target_project: target_package.project.name,
+           target_package: target_package.name,
+           source_project: source_package.project.name,
+           source_package: source_package.name)
+  end
+
+  before do
+    login(user)
+  end
+
+  describe '#create' do
+    let(:description) { Faker::Lorem.sentence }
+
+    context 'success' do
+      before do
+        post :create, params: { staging_workflow_id: staging_workflow.id, request_ignoration: { bs_request_id: bs_request.number, description: description } }
```

Comment:
`staging_workflow: staging_workflow` :see_no_evil:

Reply:
It seems that `staging_workflow_id` is still needed. So changed to `staging_workflow_id: staging_workflow`.